src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -181,6 +181,8 @@ add_executable(
   test_concurrency_manager.cc
   test_custom_load_manager.cc
   test_sequence_manager.cc
+  mock_sequence_manager.h
```

Comment:
nit: move `mock_sequence_manager` near the `mock_data_loader`.

Reply:
Moved 👍